docs/snippets/cpu/multi_threading.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <openvino/runtime/core.hpp>
+
+#include "openvino/runtime/intel_cpu/properties.hpp"
+
+int main() {
+    try {
+        std::string modelPath = "model.xml";
+        std::string device = "CPU";
+        ov::AnyMap config;
+        ov::Core core;
+        core.set_property(ov::inference_num_threads(1));
+        auto model = core.read_model(modelPath);
+        //! [ov:intel_cpu:multi_threading:part0]
+        // Use one processor for inference
```

Comment:
`processor` or `core`?

Reply:
The main concern is terminology. `Processor` usually refers to the whole CPU as a block. So maybe `logical core` or `logical processor` better explains the idea. This is applicable to each invocation of `processor`.

Reply:
updated